### DISCUSSION TITLE
Not Found error tried every method(docker run and docker compose)

### DISCUSSION THREAD

**Discussion by Greatz08** [gotenberg/gotenberg#738] [Q&A] [answered]
docker run --rm -p 4800:3000 gotenberg/gotenberg:7  used this command and got error and modified host port only in docker compose and tried to create but got same error :-(
guide please if i am doing something wrong. error 500 for every try to use api
![20231206_17h04m43s_grim](https://github.com/gotenberg/gotenberg/assets/55040435/12118e23-ea5d-4aa0-9f0b-8d66fa167d1f)
![20231206_17h01m47s_grim](https://github.com/gotenberg/gotenberg/assets/55040435/e5d36e10-e45b-4f46-a503-2d5905169ab6)

**Comment by gulien** [comment]
Hello @shwetkanwal24,

That's not a 500, but a 404. You're trying to use the API through your browser, but it won't work. Try to use cURL or Postman or any clients capable of creating form data requests.

**Comment by Greatz08** [comment]
@gulien oops i attached different image btw i also know status 404 means not found, i rechecked my ss section and i didnt take the ss of 500 error otherwise i would have attached that anyways fixed that issue, i do know that this is api url so wasnt expecting and web interface, i started getting this error 404 error so wanted to highlight this error majorly. i did recreate and run the curl command to convert things to pdf and i am getting timeout error 
![20231207_08h14m52s_grim](https://github.com/gotenberg/gotenberg/assets/55040435/5f7647cb-ad20-4e03-b473-3463f73a2731)

**Reply by gulien** [reply, marked as answer]
Try increasing the Chromium start timeout with the flag `--chromium-start-timeout=30s` 🤔

**Reply by Greatz08** [reply]
thanks for guiding dev, now things are working fine
thanks again for this awesome project :-)
